website/docs/extensions/testing.md

```diff
@@ -48,9 +60,9 @@ test = "scarb cairo-test && pytest"
 It is also possible to run multiple Cairo test runners, if there is a sensible use case for this.
 Mind, that you will probably want to conditionally disable tests depending on the running test runner, so that tests
 will not be executed multiple times.
-For example, to make `scarb test` run `scarb cairo-test` and then Starknet Forge in a sequence, type the following:
+For example, to make `scarb test` run `scarb cairo-test` and then Starknet Foundry in a sequence, type the following:
 
 ```toml
 [scripts]
-test = "scarb cairo-test && snforge"
+test = "scarb cairo-test && snforge test"
 ```
```

Comment:
Just remove this example imo, it makes completely no sense. It won't work. pytest example is a good one and it is sufficient.